src/main/java/com/google/opensesame/projects/ProjectDatastore.java

```diff
@@ -19,8 +24,17 @@
    * @return Returns the key associated with the datastore entity.
    */
   public static Key addProjectEntity(ProjectEntity projectEntity) {
+    if (projectEntity.getNumMentors() <= 0) {
+      System.out.println(
```

Comment:
Is this intended to get to production? If not, please add a TODO to remember to remove it

Reply:
Good point. I'll be making the switch to objectify momentarily so it'll be gone then!